source/util/CommandLine.cpp

```diff
@@ -801,4 +865,32 @@ std::string CommandLine::Option::set(OptionCallback& target, string_view, string
     return target(value);
 }
 
+std::string CommandLine::Option::set(std::map<std::string, int>& target, string_view name,
+                                     string_view value) {
+    size_t equalsIndex = value.find_first_of(',');
+    string_view num_args;
+    if (equalsIndex != string_view::npos) {
+        num_args = value.substr(equalsIndex + 1);
```

Comment:
Noticed this in a few places in this diff, but please be consistent about naming style. Local variables are camelCase, no underscores.

Reply:
I'll fix variable naming.